### ISSUE TITLE
NotFound not triggered.

### ISSUE THREAD

**Issue by fooman** [magento-research/code-samples#1]
When using the sandbox I was not able to get to display the 404 page. I am guessing it's missing something in the upward.yml file.

To reproduce open sandbox
![image](https://user-images.githubusercontent.com/455508/60859449-d5de7280-a266-11e9-8ae5-f0cee97cdf31.png)

change url to something that should be missing and hit enter, observe that the page does not include "Page not found!"
![image](https://user-images.githubusercontent.com/455508/60859480-f4dd0480-a266-11e9-97a4-22651500f69b.png)

**Comment by jcalcaben** [comment]
@fooman Good catch! This project started using the steps in the [hello upward](https://magento-research.github.io/pwa-studio/tutorials/hello-upward/simple-server/) tutorial, so that path is still in the upward file but is no longer used in the main application.

I'm open to any PR solution of bringing this path back though.
